### ISSUE TITLE
Warning Shown for Ignored Files

### ISSUE THREAD

**Issue by kevinastone** [SublimeLinter/SublimeLinter-eslint#157]
SublimeLinter-eslint is showing a generic linter warning on ignored files.

I'm ignoring this file intentionally so eslint doesn't bother (it's a data file in this case).

#### `SublimeLinter` console output

```
SublimeLinter: eslint: nested_arguments.js ['$REPO/node_modules/.bin/eslint', '--format', 'compact', '--stdin', '--stdin-filename', '@'] 
SublimeLinter: eslint output:
$REPO/tests/fixtures/nested_arguments.js: line 0, col 0, Warning - File ignored because of a matching ignore pattern. Use "--no-ignore" to override.

1 problem 
```

#### `eslint` output from command line

```
kstone@Kevins-MacBook-Pro $REPO> ./node_modules/.bin/eslint tests/fixtures/nested_arguments.js 

$REPO/tests/fixtures/nested_arguments.js
  0:0  warning  File ignored because of a matching ignore pattern. Use "--no-ignore" to override

✖ 1 problem (0 errors, 1 warning)
```

**Comment by kaste** [comment]
Do you care the internal console log or are there actually highlights, gutter marks drawn?

(Ah, hi, the plugin was unmaintained last year. It is now in the SublimeLinter org.)

**Comment by kaste** [comment]
Okay, got it, will be fixed via #195
